sys/include/net/ng_netopt.h

```diff
@@ -137,6 +137,11 @@ typedef enum {
     NG_NETOPT_TX_END_IRQ,
     NG_NETOPT_AUTOCCA,              /**< en/disable to check automatically
                                      *   before sending the channel is clear. */
+
+
+    NG_NETOPT_ACCESS_ADDRESS,     /**< get/set BLE access address */
```

Comment:
Since other devices could potentially use this, I would be happier with a more generic documentation. If you don't think that other device types can make use of the address (for now) I think adding `BLE_` after `NG_NETOPT_` to the value's name is best.